### ISSUE TITLE
Classic Controller / Wiimote + Nunchuk Suggestion

### ISSUE THREAD

**Issue by cheatfreak47** [FIX94/Nintendont#402]
People often complain that SSX Tricky when played with a classic controller can cause the game to reset unintentionally. For Real GC pads, there's the option of using Native Controller to remedy this, but there's no fix for if you use a Classic Controller. 
 
Here's a pretty simple solution: Disable those shortcuts outright when using a CC or CCPro, and opt for using the existing Home Button for Exit, and use L + Home for reset.

This would solve the SSX Tricky issue mentioned before, plus it just makes a lot more sense anyway (There's no need for two shortcuts to be tied the same action on one controller)

For Wiimote + Nunchuck have it be 2 + Home to Reset or something similar.

**Comment by carnage702** [comment]
CC has no exit combo without being the home button lol.

the only problem with ssx tricky is using the wiiu gcadapter where you cant use native, all others controllers have workarounds.

**Comment by cheatfreak47** [comment]
Incorrect, if you press RZ + R + B + D-pad Down, it exits, same for RZ + R + Plus
The button combos that affect the real GC pad are also present on the CC and CCPro.

**Comment by fraser125** [comment]
I believe my situation is similar, but the result is possibly more annoying (intentionally, Thanks N)

When playing GC Animal Crossing with my 3 & 4 year old on the Classic or Classic Pro controller it is VERY easy for them to bump the "blue" button (home) and reset the game. Not only do they loose the days progress but next time they play, Mr. Reseti gives them a 5 minute lecture that I have to read to them and cuts into their already limited play time. 

If the home button on the classic controller could simply be ignored and use the "home" button on the WiiMote, this problem would be solved for me. I think it's a good and reasonable default but if you want to make it another configuration option I'm fine with that too. 

> CC has no exit combo without being the home button lol.

Is the CC's 30" cable too long to reach the other "home" button? 
Or is there some GC or other emulated game that requires split second timing of the Home Button?
I feel like I'm missing something here. 
Thanks for considering!

**Comment by fraser125** [comment]
Well I've been reading "issues" similar issues for this repository and it looks like the answer is to create your own build. After reading the code I can understand why. I believe the fix would need to be made in one or both of the following areas. 
[if(BTPad[chan].button & WM_BUTTON_HOME)
				goto DoExit;](https://github.com/FIX94/Nintendont/blob/35fb6025e3d4ccb161c22576d3f62054ad8f36ac/loader/source/ppc/PADReadGC/source/PADReadGC.c#L1382)

[if(BTPad[chan].button & BT_BUTTON_HOME)
				goto DoExit;](https://github.com/FIX94/Nintendont/blob/35fb6025e3d4ccb161c22576d3f62054ad8f36ac/loader/source/ppc/PADReadGC/source/PADReadGC.c#L1422)

This happens to be a 1400 line function which supports all of the controller types for Bluetooth, USB HID and any other supported controller types. There is a lot of code here and there is a high risk of breaking one of the controller types while fixing the one you want.

I would say the best way to add this input customization functionality officially would probably be to support the USB HID INI format for the Bluetooth and I2C controllers, I'm not sure how easy/hard this would be. I'll keep looking.

**Comment by fraser125** [comment]
For anyone else wondering. 
Since I also wanted to go to the Animal Crossing Island and needed the GBA input
I'm using a Wii so my approach became use Native controller mode (required for the GBA) with 2 extension cables to the GameCube controller so the kids can pull and twist the cables, without pulling on the Wii. 
This has the benefit of removing the Home button problem entirely.

If I had stayed with the Bluetooth controllers, I would of probably unscrewed the cases and modified the Home button so it couldn't be pressed. Just a piece of wadded up paper is a simple solution.
